pkg/config/parameters.go

```diff
@@ -396,6 +397,45 @@ func (t TimeoutParameters) Validate() error {
 	return nil
 }
 
+type HeadersPolicy struct {
+	Set    map[string]string `yaml:"set,omitempty"`
+	Remove []string          `yaml:"remove,omitempty"`
+}
+
+func (h HeadersPolicy) Validate() error {
+	for key := range h.Set {
+		if msgs := validation.IsHTTPHeaderName(key); len(msgs) != 0 {
+			return fmt.Errorf("invalid header name %q: %v", key, msgs)
+		}
+	}
+	for _, val := range h.Remove {
+		if msgs := validation.IsHTTPHeaderName(val); len(msgs) != 0 {
+			return fmt.Errorf("invalid header name %q: %v", val, msgs)
+		}
+	}
+	return nil
+}
+
+// PolicyParameters holds default policy used if not explicitly set by the user
+type PolicyParameters struct {
```

Comment:
Do we think this type will be expanded upon to support more than headers? (hence the more general name?)

Reply:
Yes, exactly - anything with Policy in the name should be able to be configured here. My next target is authPolicy and possibly rateLimitPolicy after that.